### ISSUE TITLE
[Bug]: Property 'CustomsInfo' does not exist on type 'EasyPost'

### ISSUE THREAD

**Issue by magicspon** [EasyPost/easypost-node#328]
### Software Version

5.9.0

### Language Version

16.17.0

### Operating System

Mac

### What happened?

I'm trying to use the CustomsInfo api, but i'm getting a ts error saying "Property 'CustomsInfo' does not exist on type 'EasyPost'"

Is the documentation up to date, or is this type missing from the defintions.

### What was expected?

I would expect the example code from the documentation to work with typescript

### Sample Code

```node
const customsInfo = new easypost.CustomsInfo({
	eel_pfc: 'NOEEI 30.37(a)',
	customs_certify: true,
	customs_signer: 'Steve Brule',
	contents_type: 'merchandise',
	contents_explanation: '',
	restriction_type: 'none',
	restriction_comments: '',
	customs_items: [
		{
			description: 'T-shirts',
			quantity: 1,
			weight: 5,
			value: 10,
			hs_tariff_number: '123456',
			origin_country: 'US',
		},
	],
})

```
```


### Relevant logs

_No response_

**Comment by Justintime50** [comment]
Did you save the object before trying to use it? (eg: `customsInfo.save().then(console.log);`). I'm not seeing that in your snippet here.

The docs are up-to-date. You can also look at the test suite of the Node library for a working example.

After re-reading your ticket though, you mentioned this is a Typescript error only? Can you confirm (I'm not having troubles with Javascript but haven't' attempted on Typescript yet). Our Typescript definitions are in beta and may have some holes we still need to fill.

**Comment by nwithan8** [comment]
As Justin said, after constructing the local objects, you will need to run `.save()` to sync it to the API, which needs to be awaited.

`const customsInfo = await new easypost.CustomsInfo({data...}).save();`

**Comment by magicspon** [comment]
heyo, thanks for the speedy reply... still having issues with ts

![Screenshot 2023-01-13 at 17 20 18](https://user-images.githubusercontent.com/3268717/212380325-bdee224e-2c06-40af-a04b-0100c2f5e0e5.png)

**Comment by magicspon** [comment]
is this a valid way of using customs info? 

![image](https://user-images.githubusercontent.com/3268717/212380917-fc5b8e6b-2433-4079-96ff-97107ed59a58.png)

**Comment by Justintime50** [comment]
Ah, it doesn't look like we ever implemented the actual interface and tied it back to the class for `CustomsInfo` and `CustomsItem`. We'll try to get a patch out sometime next week.

**Comment by Justintime50** [comment]
These should now be implemented with the recently released `v5.10.0`!
